cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -51,49 +29,29 @@
         </div>
     {%- endfor %}
 
-    {% if prototype %}
-        <div class="block">
-            {% include 'organisms/full-width-text.html' %}
-
-            <div class="expandable-prototypes">
-                {% include 'molecules/expandable.html' with context %}
-            </div>
-
-            <div class="expandable-group-prototypes">
-                {% include 'organisms/expandable-group.html' with context %}
-            </div>
-
-        </div>
-    {% endif %}
-
-    {% if prototype %}
-        {{ related_posts.render(global_dict.related_posts_settings) }}
-        {% include 'organisms/email-signup.html' %}
-    {% else %}
-        {% for block in page.sidefoot %}
-            <div class="block block__flush-top">
-                {% if 'related_posts' in block.block_type %}
-                    {{ related_posts.render(block) }}
-                {% else %}
-                    {% if 'heading' in block.block_type %}
-                        <h3 class="o-sidebar-breakout_heading">
-                            {{ render_stream_child(block) }}
-                        </h3>
-                    {% elif 'slug' in block.block_type %}
-                        <h2 class="header-slug">
-                            <span class="header-slug_inner">
-                                {{ render_stream_child(block) }}
-                            </span>
-                        </h2>
-                    {% elif 'paragraph' in block.block_type %}
-                        {{ render_stream_child(block) }}
-                    {% else %}
+    {% for block in page.sidefoot %}
+        <div class="block block__flush-top">
+            {% if 'related_posts' in block.block_type %}
+                {{ related_posts.render(block) }}
+            {% else %}
+                {% if 'heading' in block.block_type %}
```

Comment:
Doesn't need to hold this up, but `if` statement can be combined with `else` statement above (turned into an `elif`).